samples/graph-activity-feed/csharp/README.md

```diff
@@ -0,0 +1,62 @@
+﻿# Notify Feed App
+
+Bot Framework v4 Activity Feed sample using Tab.
+
+This sample has been created using [Microsoft Graph](https://docs.microsoft.com/en-us/graph/overview?view=graph-rest-beta), it shows how trigger a Activity feed notification from your Tab, it triggers the feed notification for User, Chat and Team scope and send back to conversation.
+
+## Prerequisites
+
+- [.NET Core SDK](https://dotnet.microsoft.com/download) version 3.1
+
+  ```bash
+  # determine dotnet version
+  dotnet --version
+  ```
+
+## To try this sample
+
+- Clone the repository
+
+    ```bash
+    git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+    ```
+
+- Run the bot from a terminal or from Visual Studio:
+
+  A) From a terminal, navigate to `samples/graph-activity-feed/csharp` folder
+
+  ```bash
+  # run the bot
+  dotnet run
+  ```
+
+  B) Or from Visual Studio
+
+  - Launch Visual Studio
```

Comment:
This readme file does not talk about setting up Azure AD app which is used. Also there are other parameters in application setting which are not mentioned. 
![image](https://user-images.githubusercontent.com/31851992/115443844-68c88100-a231-11eb-9649-6a2c264d056a.png)